lib/schemas/cloudflare.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+var debug = require('debug')('forwarded')
+
+function isSecure (req) {
+  try {
+    var cf = JSON.parse(req.headers['cf-visitor'])
+    return cf.scheme !== undefined
```

Comment:
This is broken; according to https://support.cloudflare.com/hc/en-us/articles/200170536-How-do-I-redirect-HTTPS-traffic-with-Flexible-SSL-and-Apache- the value `{"scheme":"http"}` is valid and this routine would incorrectly mark the request as secure.